core-ui/src/components/Extensibility/components/CodeViewer.js

```diff
@@ -46,9 +47,10 @@ const getValueAndLang = (value, structure) => {
         parsedValue = stringifyIfObject(value);
     }
   } catch (e) {
-    window.alert(
-      `CodeViewer: received resource is in a language other than defined: ${e}`,
-    );
+    LuigiClient.uxManager().showAlert({
+      text: `CodeViewer - the received resource is in a language other than defined: ${e}`,
```

Comment:
1. Please use `useNotification`. We use luigi notifications only for notifications from `core`, where we don't have the mechanism from `core-ui`.
2. Please move the text to the translations file.